website/get-involved/index.html

```diff
@@ -280,6 +280,12 @@ <h3>{{ _('Mailing Lists') }}</h3>
                         <li>{{ _('<a href="https://wiki.mozilla.org/Thunderbird/tb-enterprise">tb-enterprise</a> -
                             For ADMINISTRATORS to discuss large scale deployment and configuration of Thunderbird.') }}
                         </li>
+                        <li>{{ _('<a href="https://thunderbird.topicbox.com/groups/addons">Add-on Developers List</a> -
+                            For general discussions about creating add-ons, and is for add-on developers who need support in creating their add-ons.') }}
+                        </li>
+                        <li>{{ _('<a href="mailto:tb-addon-reviewers@lists.thunderbird.net">Add-ons reviewers</a> -
```

Comment:
Please remove the reference to the Add-ons reviewers list, due to the comments in issue #108.

Reply:
Sure! I'm following the discussion :sweat_smile:

Reply:
I've updated the PR.